dotnet/xml/ns-Microsoft.ML.Runtime.xml

```diff
@@ -1,6 +1,5 @@
 <Namespace Name="Microsoft.ML.Runtime">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Namespace containing advanced types that support ML.NET.</summary>
```

Comment:
>Namespace containing advanced types that support ML.NET [](start = 13, length = 55)

it is mostly things related to IHost, IChannel: logging, setting the seed. 

Maybe Namespace containing types needed by the ML.NET runtime. 

Advanced types makes me thing advanced ML things.

Reply:
That sounds good to me. @shmoradims ?